php/map.jinja

```diff
@@ -64,6 +65,7 @@
         'ldap_pkg': 'php5-ldap',
         'php_ini': '/etc/php5/apache2/php.ini',
         'dev_pkg': 'php5-dev',
+        'dev_pkg': 'php5',
```

Comment:
This breaks installations on Debian as `dev_pkg` should be `php5-dev` not `php5`. I believe that this was an unintentional leftover.